app/src/assets/css/modules/_tools.scss

```diff
@@ -0,0 +1,50 @@
+// ChemProps.vue
```

Comment:
For these styles, it may be a good idea to nest them each inside of a class definition since they are being exported globally now
For example, I see that in PixelUnit.html, the outermost div is of class `pixelunit`, so putting lines 37 through 50 inside of a `pixelunit{}` class would protect that styling from overflowing to other classes

Reply:
Oh totally, thanks for pointing that out. I'll do the same on #142.